sui/deploy-contract.js

```diff
@@ -315,7 +315,9 @@ async function deploy(keypair, client, supportedContract, config, chain, options
     // Update chain configuration with deployed contract address
     chain.contracts[packageName] = {
         address: published.packageId,
-        versions: [published.packageId],
+        versions: {
+            1: published.packageId,
```

Comment:
versions start from `0`
```suggestion
            0: published.packageId,
```